lib/vpago/payway_v2/base.rb

```diff
@@ -10,6 +10,10 @@ def req_time
         @payment.created_at.strftime('%Y%m%d%H%M%S')
       end
 
+      def type
+        'pre-auth'
```

Comment:
Avoid using pre-auth as the default, as it requires additional permissions to enable. Also, include a comment with other possible values for reference.